proposal/comp110_architecture.bib

```diff
@@ -7,3 +7,91 @@ @article{shannon
 	number={3}, 
 	pages={379-423}
 }
+
+@ARTICLE{smith:launchpad, 
+author={Smith, G. and Whitehead, J. and Mateas, M. and Treanor, M. and March, J. and Mee Cha}, 
+journal={IEEE Transactions on Computational Intelligence and AI in Games}, 
+title={Launchpad: A Rhythm-Based Level Generator for 2-D Platformers}, 
+year={2011}, 
+volume={3}, 
+number={1}, 
+pages={1-16}, 
+month={March},
+}
+
+@ARTICLE{compton:platform, 
+author={Compton, Kate and Mateas, Michael}, 
+booktitle={Proceedings of the Second Artificial Intelligence and Interactive Digital Entertainment International Conference (AIIDE)}, 
+title={Procedural Level Design for Platform Games}, 
+address={Marina del Rey, CA},
+year={2006}, 
+pages={109-111}, 
+month={June},
+}
+
+
+@INPROCEEDINGS{mawhorter:occupancy, 
```

Comment:
An algorithm is described in this article, which will be useful to drawn upon. I believe, however, that this algorithm was later used in an Infinite Mario contest which would be worth looking into for further insights. :+1: